### PR TITLE
Fix bugs after ADVENT merge

### PR DESCRIPTION
**Pull request by tianyu-z**
A small patch for advent merge. Please let me know if you have any suggestions.

### CODE REVIEW

File: infer.py [hunk 1]
```diff
@@ -59,7 +59,8 @@ def parsed_args():
 def eval_folder(path_to_images, output_dir, paint=False):
     images = [path_to_images / Path(i) for i in os.listdir(path_to_images)]
     for img_path in images:
-        img = pil_image_loader(img_path, task="x", domain="val")
+        # img = pil_image_loader(img_path, task="x", domain="val")
+        img = pil_image_loader(img_path, task="x")
```
Comment (melisandeteng) [inline infer.py:63]: minor thing here but do we actually want to keep th pil image loader just for eval.py ? Or should we change everything to tensor loader?

File: shared/trainer/defaults.yaml [hunk 2]
```diff
@@ -124,14 +124,16 @@ dis:
     init_type: kaiming
     init_gain: 0.2
     use_sigmoid: false
-    num_D: 1 #Number of discriminators to use (>1 means multi-scale)
+    num_D: 3 #Number of discriminators to use (>1 means multi-scale)
```
Comment (melisandeteng) [inline shared/trainer/defaults.yaml:127]: why 3 again ?

File: infer.py (hunk 1 above)
Comment (alexrey88) [reply infer.py:63]: ahhh yeah we should probably use tensor_loader, would be a cleaner code

Comment (51N84D) [reply infer.py:63]: Agreed

File: shared/trainer/defaults.yaml (hunk 2 above)
Comment (51N84D) [reply shared/trainer/defaults.yaml:127]: Yeah, I actually think keeping 1 as default is better as well

Comment (tianyu-z) [reply shared/trainer/defaults.yaml:127]: Victor suggested 3 Lol. Okay, I will change it back to 1.

**Pull request closed without merging**